meteor-app/server/api/v1/accounts/methods.ts

```diff
@@ -50,6 +55,9 @@ Meteor.methods({
 
 	// used in api, creates a viewer user under an account
 	Create_Viewer_User: function (jwt, json) {
+		check(jwt, String)
+		check(json, {username : String, emails : [String], 
```

Comment:
We should only support 1 email per user for the time being, so refactor the emails: [string]

to just support : string